src/jsonUtils/computeNode.js

```diff
@@ -0,0 +1,457 @@
+      }
+      if (alignContent === 'flex-end') {
+        yogaNode.setAlignContent(yoga.ALIGN_FLEX_END);
+      }
+      if (alignContent === 'center') {
+        yogaNode.setAlignContent(yoga.ALIGN_CENTER);
+      }
+      if (alignContent === 'stretch') {
+        yogaNode.setAlignContent(yoga.ALIGN_STRETCH);
+      }
+      if (alignContent === 'baseline') {
+        yogaNode.setAlignContent(yoga.ALIGN_BASELINE);
+      }
+      if (alignContent === 'space-between') {
+        yogaNode.setAlignContent(yoga.ALIGN_SPACE_BETWEEN);
+      }
+      if (alignContent === 'space-around') {
+        yogaNode.setAlignContent(yoga.ALIGN_SPACE_AROUND);
+      }
+      if (alignContent === 'auto') {
+        yogaNode.setAlignContent(yoga.ALIGN_AUTO);
+      }
+    }
+
+    // Align Items
+    const alignItems = style.alignItems;
+    if (alignItems) {
+      if (alignItems === 'flex-start') {
+        yogaNode.setAlignItems(yoga.ALIGN_FLEX_START);
+      }
+      if (alignItems === 'flex-end') {
+        yogaNode.setAlignItems(yoga.ALIGN_FLEX_END);
+      }
+      if (alignItems === 'center') {
+        yogaNode.setAlignItems(yoga.ALIGN_CENTER);
+      }
+      if (alignItems === 'stretch') {
+        yogaNode.setAlignItems(yoga.ALIGN_STRETCH);
+      }
+      if (alignItems === 'baseline') {
+        yogaNode.setAlignItems(yoga.ALIGN_BASELINE);
+      }
+    }
+
+    // Align Self
+    const alignSelf = style.alignSelf;
+    if (alignSelf) {
+      if (alignSelf === 'flex-start') {
+        yogaNode.setAlignSelf(yoga.ALIGN_FLEX_END);
+      }
+      if (alignSelf === 'flex-end') {
+        yogaNode.setAlignSelf(yoga.ALIGN_FLEX_END);
+      }
+      if (alignSelf === 'center') {
+        yogaNode.setAlignSelf(yoga.ALIGN_CENTER);
+      }
+      if (alignSelf === 'stretch') {
+        yogaNode.setAlignSelf(yoga.ALIGN_STRETCH);
+      }
+      if (alignSelf === 'baseline') {
+        yogaNode.setAlignSelf(yoga.ALIGN_BASELINE);
+      }
+    }
+
+    // Flex Wrap
+    const flexWrap = style.flexWrap;
+    if (flexWrap) {
+      if (flexWrap === 'nowrap') {
+        yogaNode.setFlexWrap(yoga.WRAP_NO_WRAP);
+      }
+      if (flexWrap === 'wrap') {
+        yogaNode.setFlexWrap(yoga.WRAP_WRAP);
+      }
+      if (flexWrap === 'wrap-reverse') {
+        yogaNode.setFlexWrap(yoga.WRAP_WRAP_REVERSE);
+      }
+    }
+  }
+
+  // if Text node
+  if (
+    node &&
+    node.type === 'text' &&
+    node.children &&
+    (typeof node.children[0] === 'string' ||
+      typeof node.children[0] === 'number')
+  ) {
+    const content = String(node.children[0]);
+    const textStyle = context.getInheritedStyles();
+    const measure = createStringMeasurer(content, textStyle)();
+
+    if (!style.height) {
+      yogaNode.setHeight(measure.height);
+    }
+
+    // Use hardcoded width if present
+    if (style.width || typeof style.width === 'number') {
+      return yogaNode;
+    }
+
+    yogaNode.setWidth(measure.width);
```

Comment:
You can use `node.setMeasureFunc` for this

Reply:
Is there any extra advantage to using `node.setMeasureFunc`?

Reply:
You’ll need it to get multi-line text working. Especially true when flex is involved, as you don’t know the width at this point.

Reply:
Fair enough. Done.

Seems like there might be a bit more work to be done within `createStringMeasurer.js` as multiline text doesn't seem to work correctly, but we're on the right track.